src/python_pybind11/src/Helpers.cc

```diff
@@ -0,0 +1,231 @@
+/*
+ * Copyright (C) 2021 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+#include <tuple>
+
+#include <pybind11/stl.h>
+
+#include "Helpers.hh"
+#include <ignition/math/Helpers.hh>
+#include <ignition/math/Vector3.hh>
+
+namespace ignition
+{
+namespace math
+{
+namespace python
+{
+class Helpers
+{
+};
+
+/// \brief Compute sphere volume
+/// \param[in] _radius Sphere radius
+/// \return sphere volume
+float SphereVolume(const float _radius)
+{
+  return IGN_SPHERE_VOLUME(_radius);
+}
+
+/// \brief Compute cylinder volume
+/// \param[in] _r Cylinder base radius
+/// \param[in] _l Cylinder length
+/// \return cylinder volume
+float CylinderVolume(const float _r, const float _l)
+{
+  return IGN_CYLINDER_VOLUME(_r, _l);
+}
+
+/// \brief Compute box volume
+/// \param[in] _x X length
+/// \param[in] _y Y length
+/// \param[in] _z Z length
+/// \return box volume
+float BoxVolume(const float _x, const float _y, const float _z)
+{
+  return IGN_BOX_VOLUME(_x, _y, _z);
+}
+
+/// \brief Compute box volume from a vector
+/// \param[in] _v Vector3d that contains the box's dimensions.
+/// \return box volume from a vector
+float BoxVolumeV(const ignition::math::Vector3d &_v)
+{
+  return IGN_BOX_VOLUME_V(_v);
+}
```

Comment:
I intend on replacing these macros with `constexpr` functions in `main`, just a thought when we are forward porting.